nvssclient/Controllers/RecordController.cs

```diff
@@ -155,12 +160,88 @@ public async Task<ActionResult> VoidRecordHandler([FromBody] List<object> textLi
         // Wraps the record in a FHIR Void message and queues the message to be sent to the NVSS API Server
         [HttpPost]
         [Route("void")]
-        public async Task<ActionResult> VoidRecordHandler([FromBody] object recordJson)
+        public async Task<ActionResult> VoidRecordHandler([FromBody] Dictionary<string,object> json)
```

Comment:
This method is declared as asynchronous but does not await anything. Either drop the `async` or use `await` to wait on an asynchronous operation (such as the database).